GCP-2948: Add region filtering docs to GCP quick start (#33617)
DataDog/documentation: content/en/getting_started/integrations/google_cloud.md

{{% /collapse-content %}}

{{% collapse-content title="Limit metric collection by Google Cloud region, and by global resources" level="h4" %}}

> Under the **Metric Collection** tab in Datadog's [Google Cloud integration page][11], deselect which region for Datadog to exclude from metrics collection.

Author: good call can update since there may be multiple regions to deselect.

> Additionally, users can specify additional locations not listed and disable any global metrics not associated with a region.

Reviewer: ```suggestion
You can also specify additional locations not listed, and disable collection of any global metrics not associated with a region.
```

{{% /collapse-content %}}

{{% collapse-content title="Limit metric collection by Google Cloud region, and by global resources" level="h4" %}}

Under the **Metric Collection** tab in Datadog's [Google Cloud integration page][11], deselect which regions to exclude from metrics collection.

You can also specify additional locations not listed and disable any global metrics not associated with a region.

> {{< img src="integrations/google_cloud_platform/metric_region_filtering.png" alt="The metric collection tab in the Datadog Google Cloud integration page, with predefined regions defined Adding custom multi-region filters" style="width:80%;">}}

Reviewer: ```suggestion
{{< img src="integrations/google_cloud_platform/metric_region_filtering.png" alt="The metric collection tab in the Datadog Google Cloud integration page, with the Enable Global Metrics option highlighted and a subset of regions selected. The Additional Locations option is also highlighted with a multi-region filter defined" style="width:80%;">}}
```

Author: Sure I can re-take the screenshot and zoom in! And no that would not, since the `europe-north` is a defined region while `europe` is a separate location used by subset of services that is **not** the direct parent of the `europe-north` zone. Google is weird in that these are used inconsistently despite the fact that the naming implies a lineage between regions and locations.

Author: But I can also make the example a little less confusing here
